pkg/common/adaptor/response.go

```diff
@@ -88,3 +92,68 @@ func GetCompatResponseWriter(resp *protocol.Response) http.ResponseWriter {
 	c.header = h
 	return c
 }
+
+// NewHertzHTTPHandlerFunc wraps net/http handler to hertz app.HandlerFunc,
+// so it can be passed to hertz server
+//
+// While this function may be used for easy switching from net/http to hertz,
+// it has the following drawbacks comparing to using manually written hertz
+// request handler:
+//
+//   - net/http -> hertz handler conversion has some overhead,
+//     so the returned handler will be always slower than manually written
+//     hertz handler.
+//
+// So it is advisable using this function only for quick net/http -> hertz switching.
+// Then manually convert net/http handlers to hertz handlers
+func NewHertzHTTPHandlerFunc(h http.HandlerFunc) app.HandlerFunc {
```

Comment:
It seems inappropriate to put it in `response.go`, create hanlder.go and handler_test.go